modules/solid_mechanics/src/materials/ADComputeIncrementalShellStrain.C

```diff
@@ -41,6 +41,9 @@ ADComputeIncrementalShellStrain::validParams()
                                        "Quadrature order in out of plane direction");
   params.addParam<bool>(
       "large_strain", false, "Set to true to turn on finite strain calculations.");
+  params.addParam<RealVectorValue>(
+      "first_local_vector_ref",
+      "Optional vector defining the reference direction for the first local vector e1");
```

Comment:
I updated the description as you suggested, but I don't want to default the variable to RealVectorValue(0, 1, 0) because the local coordinate computation is handled differently when the user defines the "reference_first_local_direction". To clarify this, I added more details to the .md file. I hope, my explaination is clear!